docs/about/features_index/fed_eval.rst

```diff
@@ -24,34 +24,489 @@ In general pipeline is as follows:
+                                aggregator.py:409
 
-.. literalinclude:: ../../../openfl-workspace/torch_cnn_mnist_fed_eval/plan/plan.yaml
+    ==> collab2.log <==
+            INFO     Received Tasks: [name: "aggregated_model_validation"                                           
+                            collaborator.py:184
+                        , name: "train"                                                                                
 
-.. literalinclude:: ../../../openfl-workspace/workspace/plan/defaults/federated-evaluation/aggregator.yaml
+                        , name: "locally_tuned_model_validation"                                                       
 
-.. literalinclude:: ../../../openfl-workspace/workspace/plan/defaults/federated-evaluation/assigner.yaml
+                        ]                                                                                              
 
-.. literalinclude:: ../../../openfl-workspace/workspace/plan/defaults/federated-evaluation/tasks_torch.yaml
+Post the end of learning federation we can note what is the best model accuracy reported and save the best.pbuf file for next step - evaluation
 
-Key Changes for Federated Evaluation by baking in defaults for:
+.. code-block:: shell
+
+        ==> fx_aggregator.log <==
+    [06:09:27] INFO     Collaborator collaborator1 is sending task results for train, round 1                          
+            
+    [06:09:28] INFO     Collaborator collaborator1 is sending task results for locally_tuned_model_validation, round 1                             aggregator.py:629
+            INFO     Round 1: Collaborators that have completed all tasks: ['collaborator2', 'collaborator1']                                  aggregator.py:1049
+            INFO     Round 1: saved the best model with score 0.960096                                              
+            
+            INFO     Saving round 1 model...                                                                        
+
+            INFO     Experiment Completed. Cleaning up...                                                           
+
+In this case we can confirm that post the 2 rounds of training the model reported an accuracy of 0.960096
+
+.. code-block:: shell
+
+    Round 1: saved the best model with score 0.960096                                              
+                                aggregator.py:955
+
+Lets save this model (best.pbuf) for later usage
+
+.. code-block:: shell
 
-1. **aggregator.settings.rounds_to_train**: Set to 1
-2. **assigner**: Assign to aggregated_model_validation instead of default assignments
-3. **tasks**: Set to aggregated_model_validation instead of default tasks
+    cp cnn_train_eval/save/best.pbuf ~/trained_model.pbuf
+    devuser@devvm:~/src/clean/openfl$ 
+
+Now lets create another workspace using the same plan using same steps as mentioned in learning Setup:

```

Comment:
typo - let's. Also, there's a repetition of "same plan... same steps". I'd suggest simply writing "same plan and steps"